webui/react/src/pages/Admin/UserManagement.tsx

```diff
@@ -226,6 +240,46 @@ const UserManagement: React.FC = () => {
     [handleNameSearchApply, handleNameSearchReset, settings.name],
   );
 
+  const handleTableRowSelect = useCallback((rowKeys: React.Key[]) => {
+    setSelectedUserIds(rowKeys);
+  }, []);
+
+  const actionDropdownMenu: MenuItem[] = useMemo(() => {
+    const menuItems: MenuItem[] = [{ key: ActionMenuKey.ChangeStatus, label: 'Change Status' }];
+
+    if (rbacEnabled) {
+      if (canModifyPermissions) {
+        menuItems.push({ key: ActionMenuKey.SetRoles, label: 'Set Roles' });
+      }
+      if (canModifyUsers) {
+        menuItems.push({ key: ActionMenuKey.AddToGroups, label: 'Add to Groups' });
```

Comment:
are these permissions correct?

Reply:
@gt2345 do you know if its correct?

Reply:
LGTM, these permissions are align with setting group/role individually